crates/types/src/bounding_box.rs

```diff
@@ -0,0 +1,26 @@
+use coordinate_systems::Pixel;
+use geometry::rectangle::Rectangle;
+use serde::{Deserialize, Serialize};
+use serialize_hierarchy::SerializeHierarchy;
+
+#[derive(Debug, Clone, Copy, Serialize, Deserialize, SerializeHierarchy)]
+pub struct BoundingBox {
```

Comment:
This is not only a bounding box: I do not like the name. I miss the scoring functionality

Reply:
If the context of a yolo bounding box this makes sense.